src/os_view.eliom

```diff
@@ -3,10 +3,25 @@
    and integrate it in your app to customize the behaviour according to
    your needs. *)
 
+(** This module defines functions to create password forms, connection forms,
+ * settings buttons and other stuff common contents to easily create content
+ * and start your application. Feel free to add other contents you need.
+ * As Eliom_content.Html.F is opened by default, if the module D is not
+ * explicitly used, HTML tags will be functional.
+ *)
```

Comment:
It sounds like this comment is addressed to OS developers, not end users. Please use `(**` and `(*` consistently to distinguish API doc from regular comments.